app/src/main/kotlin/cz/covid19cz/erouska/ui/main/MainActivity.kt

```diff
@@ -1,36 +1,55 @@
 package cz.covid19cz.erouska.ui.main
 
 import android.content.*
+import android.net.Uri
 import android.os.Bundle
+import android.os.Handler
 import android.view.MenuItem
 import android.view.View.GONE
 import android.view.View.VISIBLE
 import androidx.browser.customtabs.CustomTabsClient
 import androidx.browser.customtabs.CustomTabsServiceConnection
 import androidx.core.content.ContextCompat
+import androidx.core.content.pm.ShortcutInfoCompat
+import androidx.core.content.pm.ShortcutManagerCompat
+import androidx.core.graphics.drawable.IconCompat
 import androidx.lifecycle.Observer
 import androidx.localbroadcastmanager.content.LocalBroadcastManager
 import androidx.navigation.NavDestination
 import androidx.navigation.NavOptions
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
+import cz.covid19cz.erouska.BuildConfig
 import cz.covid19cz.erouska.R
 import cz.covid19cz.erouska.bt.BluetoothRepository
 import cz.covid19cz.erouska.databinding.ActivityMainBinding
 import cz.covid19cz.erouska.ext.hasLocationPermission
 import cz.covid19cz.erouska.ext.isLocationEnabled
+import cz.covid19cz.erouska.receiver.ActionsReceiver
 import cz.covid19cz.erouska.service.CovidService
 import cz.covid19cz.erouska.ui.base.BaseActivity
 import cz.covid19cz.erouska.utils.CustomTabHelper
+import cz.covid19cz.erouska.utils.L
 import kotlinx.android.synthetic.main.activity_main.*
 import org.koin.android.ext.android.inject
 
+
 class MainActivity :
     BaseActivity<ActivityMainBinding, MainVM>(R.layout.activity_main, MainVM::class) {
 
+    companion object {
+        object ShortcutsActions {
+            private const val SCHEME = "erouska"
+
+            val URL_PAUSE: Uri = Uri.parse("${SCHEME}://${BuildConfig.FLAVOR}/pause")
```

Comment:
I would put flavor into scheme, better PROD will have only `erouska://` DEV will have `erouska-dev://`

Reply:
Sorry, I forgot to mention this in "things to discuss" :-) I'll change the scheme and also the broadcasted intent.